alternator/ttl.cc

```diff
@@ -72,36 +72,31 @@ future<executor::request_return_type> executor::update_time_to_live(client_state
     }
 
```

Comment:
> alternator: Add support for TTL when using tablets

The changelog directly dives into the details but doesn't give the high level overview. What was the problem with tablets and how was it solved?